ghcid.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version:      >= 1.18
 build-type:         Simple
 name:               ghcid
-version:            0.6.10
+version:            0.6.11
```

Comment:
As a note for next time (but not really a big deal either way) I only update the version number when I make a release, as per the last bullet at https://github.com/ndmitchell/neil#contributions